app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -433,6 +437,36 @@ class AdministratorControlsFragmentTest {
     }
   }
 
+  @Test
+  fun testFragmentArguments_afterCreation_areCorrect() {
+    launch<AdministratorControlsFragmentTestActivity>(
+      createAdministratorControlsFragmentTestActivityIntent(
+        profileId = internalProfileId
+      )
+    ).use { scenario ->
+      testCoroutineDispatchers.runCurrent()
+      scenario.onActivity { activity ->
+
+        val administratorControlsFragment = activity.supportFragmentManager
+          .findFragmentById(R.id.administrator_controls_fragment_test_activity_fragment_container)
+          as AdministratorControlsFragment
+        val isMultipane = activity
+          .findViewById<DrawerLayout>(R.id.administrator_controls_activity_drawer_layout) != null
+
+        val arguments = checkNotNull(administratorControlsFragment.arguments) {
+          "Expected arguments to be passed to AdministratorControlsFragment"
+        }
+        val args = arguments.getProto(
+          ADMINISTRATOR_CONTROLS_FRAGMENT_ARGUMENTS_KEY,
+          AdministratorControlsFragmentArguments.getDefaultInstance()
+        )
+        val receivedIsMultipane = args.isMultipane
```

Comment:
Can be simplified to:
```suggestion
  val arguments = checkNotNull(administratorControlsFragment.arguments) {
        "Expected arguments to be passed to AdministratorControlsFragment"
      }.getProto(
        ADMINISTRATOR_CONTROLS_FRAGMENT_ARGUMENTS_KEY,
        AdministratorControlsFragmentArguments.getDefaultInstance()
      )
      val receivedIsMultipane = arguments.isMultipane
```

Reply:
Done.